packages/back-end/src/services/features.ts

```diff
@@ -44,6 +50,66 @@ function generatePayload(
   return defs;
 }
 
+export type VisualExperiment = {
+  experiment: ExperimentInterface;
+  visualChangeset: VisualChangesetInterface;
+};
+
+// TODO Put this in a better place?
```

Comment:
You could have a new file `./services/visual-experiments.ts` and add all utils there but if visual experiments are tightly coupled to features, this might be ok, though they seem more related to experiments.

Reply:
Ok, thanks. Maybe @jdorn  has thoughts? I'll remove the comment if no one has strong suggestions...

Reply:
No strong opinion, you could argue both sides.
1. generateVisualExperimentsPayload serves the purpose of populating the Features SDK payload, thus features
2. it could grow in complexity and in utility functions, thus should be its own service

Maybe leave it alone until we actually add more visual exp service functions?

Reply:
It seems more related to experiments but I don't have strong feelings either way.